modelling/src/main/java/org/axonframework/modelling/command/AggregateAlreadyExistsException.java

```diff
@@ -0,0 +1,34 @@
+package org.axonframework.modelling.command;
+
+import org.axonframework.common.AxonNonTransientException;
+
+/**
+ * Exception indicating that concurrent access to a repository was detected. Most likely, two threads were creating the
+ * same aggregate.
+ *
+ * @author Lucas Campos
+ * @since 4.3
+ */
+public class AggregateAlreadyExistsException extends AxonNonTransientException {
```

Comment:
I think we should rename the exception to `AggregateIdentifierAlreadyExistsException`

Reply:
I'd also change the javadoc accordingly to be specific it's not about an exact aggregate type which has been duplicated, but purely the identifier which is required to be unique.